src/js/providers/utils/quality-labels.ts

```diff
@@ -32,29 +44,31 @@ export function createLabel(height, bandwidth, redundant) {
 
 // Ensures that we're able to find a custom label. As long as there is at least 1 quality label and a defined
 // bandwidth, a quality label will always be found. Return null otherwise
-export function getCustomLabel(qualityLabels, bandwidth) {
-    let label = null;
+export function getCustomLabel(qualityLabels: QualityLabels | null, bandwidth?: number): string | null {
+    let label: string | null = null;
 
     if (bandwidth && qualityLabels) {
         const bandwidths = Object.keys(qualityLabels);
-        const key = parseFloat(bandwidth);
```

Comment:
qq - is this function used elsewhere in non-typed files that don't pass bandwidth in as a number? that could break without this `parseFloat`

Reply:
This function is only called by `generateLabel` or in unit tests. For the unit tests, bandwidth is always a number. I moved the `parseFloat` to `generateLabel` so the bandwidth passed from that will always be a number as well.

Reply:
What was the eventual resolution here? just wanna be sure i'm not missing anything

Reply:
I think it's safe to assume that bandwidth/bitrate will be a number based on the `level` types that are passed into `generateLabel`. Those types are the direct hls/shaka/flash provider levels and the type definitions are outlined here:
https://github.com/jwplayer/jwplayer/pull/3657#discussion_r408314156
For hls, `bitrate` is a number and same for shaka with `bandwidth`